docs/structured-streaming-programming-guide.md

```diff
@@ -930,7 +930,7 @@ There are a few types of output modes.
 new rows added to the Result Table since the last trigger will be 
 outputted to the sink. This is supported for only those queries where 
 rows added to the Result Table is never going to change. Hence, this mode 
-guarantees that each row will be output only once (assuming 
+guarantees that each row will be outputted only once (assuming 
```

Comment:
Not a typo, I agree, but I thought it made sense to keep the same formulation as what is used in the same paragraph